grpc-client-utils/build.gradle.kts

```diff
@@ -6,19 +6,11 @@ plugins {
 }
 
 dependencies {
-  implementation(project(":grpc-context-utils"))
+  api("io.grpc:grpc-context:1.40.0")
```

Comment:
also did some dependency cleanup. Could potentially have downstream affects, but only if the consumers are misconfigured.